controllers/drplacementcontrol_controller.go

```diff
@@ -976,6 +977,14 @@ func (r *DRPlacementControlReconciler) createDRPCInstance(
 		log.Info("volsync is set to disabled")
 	}
 
+	if !ramenConfig.VolSync.Disabled && drpcInAdminNamespace(drpc, ramenConfig) {
```

Comment:
If it is a metro setup, we would have set `volSyncDisabled` in line 974, which MAY get overwritten due to the env flag below as we check `ramenConfig` and not decisions already made above. While it can be argued that volsync feature should not be enabled in a MetroDR setup, it may so happen that there are multiple DR clusters and policies in play.

I would suggest replacing this with `!d.volSyncDisabled` instead.

Reply:
Done